README.md

```diff
@@ -286,6 +286,10 @@ In the same manner you can customize the button labels:
 - `craue_formflow_button_label_back` for the __back__ button
 - `craue_formflow_button_label_reset` for the __reset__ button 
 
+In the same mannero you can also remove the reset button :
+
+- `craue_formflow_render_reset_button` This option have a default value to __true__
```

Comment:
I'd prefer renaming the parameter to `craue_formflow_button_render_reset` for consistency with `craue_formflow_button_class_reset` and `craue_formflow_button_label_reset`.

How about this instead...?

> You can also remove the reset button by setting `craue_formflow_button_render_reset` to `false`.

And would you move that below the following example block? It fits better there.